bibliopixel/animation/collection.py

```diff
@@ -25,9 +27,12 @@ def pre_run(self):
 
     @property
     def current_animation(self):
-        return self.animations[self.index]
+        if self.index >= 0:
+            return self.animations[self.index]
+        else:
+            return None
```

Comment:
This doesn't work for indices greater than the number of animations!

Why not return `self.animations.get(self.index)`?